### PR TITLE
Debian build

### PR DESCRIPTION
**Pull request by allenyin96**
Modified gradle build files to include Debian build.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: alerting/build.gradle [hunk 1]
```diff
@@ -120,4 +120,4 @@ run {
     distribution = "oss-zip"
 }
 
-apply from: '../build-tools/rpmbuild.gradle'
+apply from: '../build-tools/pkgbuild.gradle'
```
Comment (mihirsoni) [inline alerting/build.gradle]: I don't see this file renamed in the diff.

Comment (allenyin96) [reply alerting/build.gradle]: It says "build-tools/rpmbuild.gradle → build-tools/pkgbuild.gradle" for me in the title of the second diff on this page. Is this what you're looking for?

**Comment by dbbaughe** [comment]
Hi Allen,

For future reference please fork the repository for PRs instead of pushing your branch up to the repository. Only release branches and long-lived feature branches that we are looking for comments on should be pushed up.

**Comment by allenyin96** [comment]
Hi Drew,

Thanks for the heads-up. Will do in the future.

**Pull request merged**